ui/app/components/auth/login-form.js

```diff
@@ -73,7 +73,7 @@ export default class AuthLoginFormComponent extends Component {
       if (data?.mfa_requirement) {
         const parsedMfaAuthResponse = this.auth._parseMfaResponse(data.mfa_requirement);
         // calls onAuthResponse in parent auth/page.js component
-        this.args.onSuccess(parsedMfaAuthResponse, backendType, data);
```

Comment:
I'm still referencing old components and tests as I wire things up, so updated here too to double check changing these args didn't break everything.